### ISSUE TITLE
If a favorite store has been changed, it will no longer be available in favorites

### ISSUE THREAD

**Issue by seluianova** [digitalfabrik/entitlementcard#1493]
**Is your feature request related to a problem? Please describe.**
Because of the current implementation of data import, when we change any attributes of the acceptance store, we delete the old record in the database and create a new one with a new ID.
As a side effect, if a modified store has been saved as a favorite on some device, it will appear as "No longer available" to the user. 
The user will have to delete it from favorites and save it again.

**Describe the solution you'd like**
Would be nice to get constant acceptance store ids from the upstream, instead of generating them on our side.

**Describe alternatives you've considered**
Perhaps we could treat some of the store attributes as constant (like name and coordinates?) and use them to compare data during the import.

**Additional context**

**Comment by f1sh1918** [comment]
@seluianova i think this should be prio high, or we should disable the favorites by a feature flag until we fixed that. Should be discussed in the next weekly

**Comment by f1sh1918** [comment]
I checked the xml file and we get for about 60% of stores an id (freinet) and for the all the LBE stores we don't get any ID.
I think the ids are still missing for the 40% of the stores from ZBFS

**Comment by f1sh1918** [comment]
Yes in general that would also be my idea. The issue is until now that we can not 100% match the region but maybe 95%
